### PR TITLE
fix: separate glob for watching

### LINKED ISSUE #19: 0.6.7 and newer aren't finding components on Windows
(Haven't yet tested on Mac)

0.6.6 debugging of sample in Windows
```
  vite-plugin-components:glob started with: [src/components/**/*.{vue,md}] +0ms
  vite-plugin-components:glob 11 components found. +7ms
```

0.6.7 debugging
```
vite-plugin-components:glob started with: [C:\Data\Git\vite-plugin-components\example\src\components/**/*.{vue,md}] +0ms
  vite-plugin-components:glob 0 components found. +9ms 
```

Looks like the path is getting added to the wrong place.

### PR DESCRIPTION
**Pull request by JohnCampionJr**
Another try to fix #19

### CODE REVIEW

File: src/context.ts [hunk 1]
```diff
@@ -25,19 +25,19 @@ export class Context {
     public readonly viteConfig: ResolvedConfig,
   ) {
     this.options = resolveOptions(options, viteConfig)
-    const { globs, dirs } = this.options
+    const { watchGlobs, dirs } = this.options
```
Comment (antfu) [inline src/context.ts:28]: Can we just use the `globs` here? Maybe we just need to normalization to replace black slash to slash?

Comment (antfu) [reply src/context.ts:28]: Just saw your comment, I will merge this for now and seeking for a better solution when I get access to a Windows PC. Thank you!

Comment (JohnCampionJr) [reply src/context.ts:28]: Yeah, we could do the backslash replacement too.  It's kind of a choice between two messy things.  If you want me to test any other alternatives let me know; happy to help

Comment (antfu) [reply src/context.ts:28]: I think it has a lighter mental burden to "always works on slash". My try on #23 works for Mac, if you don't mind, I'd be grateful if you'd like to help on testing out. (guess it's time for me to setup e2e test then, always been too lazy to do so 😅)

**Pull request merged**